apps/client/src/intros/DakkapelIntro.js

```diff
@@ -39,7 +39,7 @@ export default () => (
       </ListItem>
     </List>
     <Paragraph>
-      <ContactSentence eventName={eventNames.INTRO_EXTRAORDINARY} />
+      <ContactSentence eventLocation={sections.INTRO} />
```

Comment:
This makes it less specific. Why not separate extraordinary from other cases?

Reply:
Because this is the only phonenumber event on the Intro page